pkg/client/clientset/versioned/doc.go

```diff
@@ -1,11 +1,11 @@
 /*
-Copyright The Kubernetes Authors.
+Copyright 2018 The Rook Authors. All rights reserved.
```

Comment:
The new versions of codegen can replace `DATE` in the template with the current year. I'd like to get @travisn 's and @jbw976 's opinions on whether we should be keeping the year up-to-date when doing codegen or not.

Reply:
good point. We could also update our header file.

Reply:
I've read different suggestions from different sources, so this topic has never been super clear to me.  But my own personal opinion is that I would only update the year/date in a license header when there are significant changes to the file content. I wouldn't update it when there are no changes, just to keep it up to date with the current year.

Would using the `DATE` template cause the date to update every time it runs when the year has changes, even if there are no other changes? 🤔

Reply:
Agreed, I wouldn't worry about changing the template either, even if there are updates to the file. We've typically  left the date alone once the file is created. So it would be nice to avoid updating all these files.

Reply:
@travisn  wrote:
> Agreed, I wouldn't worry about changing the template either, even if there are updates to the file. We've typically left the date alone once the file is created.

sounds good. That date tracking is what we have git for, anyways.


> So it would be nice to avoid updating all these files.

The commit can be removed from the PR but because the generated files changed (due to the use of a different header file), it was necessary to add the commit to have the PR pass the codegen ci check.

Reply:
The use of the correct boilerplate file is contained in #14915, so I want to close this PR once the other gets merged.